content/tutorials/build-an-application/_index.md

```diff
@@ -554,26 +553,18 @@ export default async request => {
 }
 ```
 
-The additional import in the first line of `src/handlers/webhook.js` signifies the final new file needed in our project: `src/config.js`. In this file, fill in the Slack Webhook URL that you created all the way back in the [Incoming Webhook] section of this guide:
-
-```javascript
-export const slackWebhookUrl = 'https://hooks.slack.com/services/abc123'
-```
-
-**This webhook allows developers to post directly to your Slack channel, so it should be kept secret!** In particular, add `src/config.js` to your `.gitignore` file to ensure that the file doesn't get committed into your source control, and published to GitHub:
+The constant `SLACK_WEBHOOK_URL` represents the Slack Webhook URL that you created all the way back in the "Incoming Webhook" section of this guide. **This webhook allows developers to post directly to your Slack channel, so it should be kept secret!** To use this constant inside of your codebase, you can use Wrangler's [Secrets](/tooling/wrangler/secrets) feature:
 
 ```sh
-echo "src/config.js" >> .gitignore
+wrangler secret create SLACK_WEBHOOK_URL
+Enter the secret text you'd like assigned to the variable name on the script named slack-bot-ENVIRONMENT_NAME: https://hooks.slack.com/services/abc123
```

Comment:
Should this line be outside the ```?

Reply:
good question - this is the output from wrangler secret so not in this case! it is very prose-y though so understand the confusion